battle.py

```diff
@@ -64,7 +64,7 @@ def attack(
     ) -> str:
         accuracy = random()
 
-        move = moves[move]
+        move = attacker.moves[move] # get the move from the attacker, so in the future we can have different moves for different enemies and players
```

Comment:
moves from attacker (attacker can be player or enemy, so each player can have different movesets). 
If you change the name inside move['name'], just have a folder with the same name and the attack images inside.